eva/catalog/models/column_catalog.py

```diff
@@ -23,19 +23,19 @@
 from eva.catalog.models.base_model import BaseModel
 
 
-class DataFrameColumn(BaseModel):
-    __tablename__ = "df_column"
+class ColumnCatalog(BaseModel):
+    __tablename__ = "column_catalog"
 
     _name = Column("name", String(100))
     _type = Column("type", Enum(ColumnType), default=Enum)
     _is_nullable = Column("is_nullable", Boolean, default=False)
     _array_type = Column("array_type", Enum(NdArrayType), nullable=True)
     _array_dimensions = Column("array_dimensions", String(100))
-    _metadata_id = Column("metadata_id", Integer, ForeignKey("df_metadata._row_id"))
+    _table_id = Column("table_id", Integer, ForeignKey("table_catalog._row_id"))
```

Comment:
Can we have a more specific naming here for both `table_id` and `table`? Because this model will be a table itself, I think it will be confusing for newcomers.

Reply:
Sorry, I cannot understand what you mean.

Reply:
Oh, do you mean to name it `_table_catalog_id`?

Reply:
Yes. I think it is a bit confusing which table this field refers to.

Reply:
Added doc string for clarity